.gitignore

```diff
@@ -277,4 +277,6 @@ lcov.info
 # WindowsInstaller generated files
 /WalletWasabi.WindowsInstaller/ComponentsGenerated.wxs
 /WalletWasabi.WindowsInstaller/*.wixobj
+
+# This file is created by an F# VS Code plugin
```

Comment:
http://ionide.io/ ?
https://github.com/ionide/Ionide-vscode-fsharp ?

Reply:
I think it is this one yes. @lontivero ?

Reply:
I added because there is a F# vscode extension that creates that file again and again.